README.md

```diff
@@ -15,101 +17,167 @@ This project defines core build rules for [Scala](https://www.scala-lang.org/) t
 
 ## Rules
 
-* [scala_library](docs/scala_library.md)
-* [scala_macro_library](docs/scala_macro_library.md)
-* [scala_binary](docs/scala_binary.md)
-* [scala_test](docs/scala_test.md)
-* [scala_repl](docs/scala_repl.md)
-* [scala_library_suite](docs/scala_library_suite.md)
-* [scala_test_suite](docs/scala_test_suite.md)
-* [thrift_library](docs/thrift_library.md)
-* [scala_proto_library](docs/scala_proto_library.md)
-* [scala_toolchain](docs/scala_toolchain.md)
-* [scala_import](docs/scala_import.md)
-* [scala_doc](docs/scala_doc.md)
+- [scala_library](docs/scala_library.md)
+- [scala_macro_library](docs/scala_macro_library.md)
+- [scala_binary](docs/scala_binary.md)
+- [scala_test](docs/scala_test.md)
+- [scala_repl](docs/scala_repl.md)
+- [scala_library_suite](docs/scala_library_suite.md)
+- [scala_test_suite](docs/scala_test_suite.md)
+- [thrift_library](docs/thrift_library.md)
+- [scala_proto_library](docs/scala_proto_library.md)
+- [scala_toolchain](docs/scala_toolchain.md)
+- [scala_import](docs/scala_import.md)
+- [scala_doc](docs/scala_doc.md)
 
 ## Getting started
 
-1. [Install Bazel](https://docs.bazel.build/versions/master/install.html), 
-see the [compatibility table](#bazel-compatible-versions).
-2. Add the following to your `WORKSPACE` file and update versions with their sha256s if needed:
+[Install Bazel][], preferably using the [Bazelisk][] wrapper. See the
+[compatbile Bazel versions](#compatible-bazel-versions) section to select a suitable
+Bazel version.
+
+[Install Bazel]: https://docs.bazel.build/versions/master/install.html
+[Bazelisk]: https://docs.bazel.build/versions/master/install.html
+
+Add the following stanza to your `WORKSPACE` file and update versions with their
```

Comment:
Can you explain what is `stanza`?
Unfortunately it is not common word. Maybe you could replace it with `code` or `code snippet`?
I was a little bit confused about it at first :-D

Reply:
Ah, sorry. Updated in https://github.com/bazelbuild/rules_scala/pull/1703/commits/b294878b07c875f2e1ca374f11e12c88593bb74e.

Reply:
https://en.wikipedia.org/wiki/Stanza

Reply:
Seems to be used originally in the context of literature, but it is also commonly used in the context of IT for a group of lines (similarly as in literature).